values.yaml

```diff
@@ -155,11 +155,13 @@ sccEnabled: false
 workers:
   # Number of airflow celery workers in StatefulSet
   replicas: 1
-  updateStrategy:
-    strategy:
-      rollingUpdate:
-        maxSurge: "100%"
-        maxUnavailable: "50%"
+  # Update Strategy when worker is deployed as a StatefulSet
+  updateStrategy: ~
+  # Update Strategy when worker is deployed as a Deployment
+  strategy:
+    rollingUpdate:
+      maxSurge: "100%"
+      maxUnavailable: "50%"
```

Comment:
This didn't affect us currently as by default persistence is not enabled, hence we deploy a Deployment instead of Statefulset.